src/mame/drivers/coleco.cpp

```diff
@@ -485,6 +686,21 @@ ROM_END
 #define rom_dina rom_czz50
 #define rom_prsarcde rom_czz50
 
+ROM_START( bit90b3 )
+	ROM_REGION( 0x4000, "banked", 0 )
+	ROM_LOAD("bit90b3.u4", 0x0000, 0x2000, CRC(06d21fc2) SHA1(6d296b09b661babd4c2ef6993f8e768a67932388))
+	ROM_LOAD("bit90b3.u3", 0x2000, 0x2000, CRC(61fdccbb) SHA1(25cac13627c0916d3ed2b92f0b2218b405de5be4))
+	ROM_REGION( 0x10000, "maincpu", 0 )
+	ROM_LOAD("bit90b3.u2", 0x2000, 0x2000, CRC(b992b940) SHA1(c7dd96a1944fac40cbae20630f303a69de7e6313))
+ROM_END
+
+ROM_START( bit90 )
+	ROM_REGION( 0x4000, "banked", 0 )
+	ROM_LOAD("d32351e.u4", 0x0000, 0x2000, NO_DUMP) // BIT-99C1
+	ROM_LOAD("d32521e.u3", 0x2000, 0x2000, NO_DUMP) // MONITOR2
+	ROM_REGION( 0x10000, "maincpu", 0 )
+	ROM_LOAD("d32522e.u2", 0x2000, 0x2000, NO_DUMP) // MONITOR3
+ROM_END
```

Comment:
Are these actually different machines, or different ROM revisions for the same machine?  If they're ROM revisions for the same machine, they should use the BIOS mechanism for selection.